src/main/java/net/neoforged/neoforge/attachment/AttachmentInternals.java

```diff
@@ -78,7 +78,7 @@ else if (tag.isEmpty())
     /**
      * Copy some attachments to another holder.
      */
-    private static <H extends AttachmentHolder> void copyAttachments(H from, H to, Predicate<AttachmentType<?>> filter) {
+    public static <H extends AttachmentHolder> void copyAttachments(H from, H to, Predicate<AttachmentType<?>> filter) {
```

Comment:
This method should remain private, and there should be a `public` bouncer that only accepts a `ProtoChunk` as a first argument and a `LevelChunk` as a second argument. (Same as `copyStackAttachments` below).